src/Agent.Worker/Release/Artifacts/CustomArtifact.cs

```diff
@@ -130,9 +130,8 @@ public IArtifactDetails GetArtifactDetails(IExecutionContext context, AgentArtif
             else
             {
                 string resourceType = streamType;
-                var errorMessage = StringUtil.Loc("RMStreamTypeNotSupported", resourceType);
-                executionContext.Output(errorMessage);
-                throw new NotSupportedException(errorMessage);
+                var warningMessage = StringUtil.Loc("RMStreamTypeNotSupported", resourceType);
+                executionContext.Warning(warningMessage);
```

Comment:
BuildArtifact logs it as warning only, so will do the same for CustomArtifact to be consistent.